lib/js/components/Buttons/Button/Button.vue

```diff
@@ -85,7 +85,7 @@ export default defineComponent({
 			},
 		},
 		color: {
-			type: String,
+			type: [String, null],
```

Comment:
This is weird. How is it different from other optional props? I thought this isn't required when we use `default`.

Reply:
I think we're using it incorrectly in our codebase. An optional prop means it can be `undefined`, not `null`. If we want to allow `null`, we should explicitly include it in the type.

I'd prefer to stop using null as the default prop value and instead rely on JS native default, which is `undefined`.

I’m not suggesting a full-scale migration from `null` to `undefined` right now, but I will propose that the team favors `undefined` over `null` moving forward. I believe we should use `null` only when we explicitly intend to clear, void, or cancel something.

Reply:
I checked how it work exactly in vue:
```
color: {
  type: String,
  default: 'myColor',
```

In TypeScript `color` prop accepts any `string` or `undefined` and not `null`
If we pass `null` as a value it will use default value (`myColor`) in the component but TS will complain.

The trick is that it would work fine in `wnl-platform` (TS would not complain) because `strictNullChecks` are disabled there. ;) Yet... not forever :D